cmd/util.go

```diff
@@ -87,6 +87,13 @@ func handleConsole(ctx context.Context, tele *telemetry.Client, cmd *cobra.Comma
 		delayMessage = false
 	case "init":
 		// No console client created here
+	case "describe":
+		var err error
+		c, err = console.ClientFactory(ctx, &cfgPath, cfgMutator)
+		if err != nil {
+			return err
+		}
+		defer c.Client().Close()
```

Comment:
nit: why not use the console.CerateClient that will return a NullClient / Normal client based on if cfg path exists and returns a console client.